proto/api.proto

```diff
@@ -1,12 +1,13 @@
 syntax = "proto3";
 package api;
-option go_package = "github.com/bloXroute-Labs/solana-trader-api/proto/api";
+option go_package = "github.com/bloXroute-Labs/solana-trader-proto/proto/api";
```

Comment:
should we actually make this `github.com/bloXroute-Labs/solana-trader-proto/api`? otherwise there's a tiny bit of redundancy in teh package name